Dockerfile

```diff
@@ -30,4 +30,7 @@ RUN make -C /tmp/px install
 COPY nginx.conf /usr/local/openresty/nginx/conf/nginx.conf
 COPY examples/creds.json /tmp/creds.json
 
+# adjust local pxconfig.lua
+#COPY pxconfig.lua /usr/local/lib/lua/px/
```

Comment:
Commented out -- is this ncecessary?

Reply:
yes, it's a very useful line how to pass a file with the custom configurations to Lua enforcer inside docker.

Reply:
👍